docs/api_keys.md

```
@@ -0,0 +1,243 @@
+name: nataliagranato
+scopes:
+  - container_registry:read
+  - container_registry:write
+  - container_registry:admin
+created_at: "2025-05-23T10:30:00Z"
+expires_at: "2026-05-23T10:30:00Z"
+```
+
+### 4. Configurar Variáveis de Ambiente
+
+#### Para uso temporário (sessão atual):
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)